package.json

```diff
@@ -57,7 +59,7 @@
   "dependencies": {
     "@godaddy/terminus": "^4.9.0",
     "bn.js": "^5.2.0",
-    "ethers": "5.4.0",
+    "ethers": "5.4.3",
```

Comment:
Was running into a number of intermittent issues locally that updating this resolved.